EventAggregator/EventAggregator.cs

```diff
@@ -179,7 +245,29 @@ private void Call<TListener>(object message, Action<Action> marshaller)
                 _config.OnMessageNotPublishedBecauseZeroListeners(message);
             }
         }
+#if ASYNC
+        private async Task CallAsync<TListener>(object message, Func<Func<Task>, Task> marshaller)
+            where TListener : class
+        {
+            int listenerCalledCount = 0;
+            await marshaller(async () =>
+            {
+                foreach (ListenerWrapper o in _listeners.Where(o => o.Handles<TListener>() || o.HandlesMessage(message)))
+                {
+                    bool wasThisOneCalled = await o.TryHandleAsync<TListener>(message);

```

Comment:
Change to `bool wasThisOneCalled = await o.TryHandleAsync<TListener>(message).ConfigureAwait(false);`

The configure await will turn off capturing the sync context. This helps improve performance and also gives users of the library an option not to run any continuations on the UI thread if they .ConfigureAwait(false) on send message async